### PR TITLE
Focusing on the input after adding category

### PR DESCRIPTION
**Pull request by waximabbax**
For: https://github.com/ProfessionalWiki/PageApprovals/issues/114

It's a little tricky now because of our existing code and also the response we get from our POST request. It sends us the entire page. However, this script should get the job done; we can also put this JS in a separate file, although I don't see the reason to it at the moment. Please do test the code on the UI.

https://github.com/user-attachments/assets/793e6713-0551-4b58-bd95-fc62bf70f1c3

### CODE REVIEW

File: templates/ManageApprovers.mustache [hunk 1]
```diff
@@ -29,19 +29,72 @@
 							</form>
 						</div>
 					{{/categories}}
-					<form method="POST" action="" class="add-category-form">
-						<input type="text" name="category" placeholder="New Category" required>
+					<form method="POST" class="add-category-form">
+						<input type="text" name="category" placeholder="New Category" id="{{username}}category" required>
```
Comment (JeroenDeDauw) [inline templates/ManageApprovers.mustache:33]: Somewhat unrelated question: what happens when username contains a "? Is it escaped?

File: templates/ManageApprovers.mustache
Comment (malberts) [inline templates/ManageApprovers.mustache:1]: I didn't review much, but the JS here isn't following MediaWiki coding standards. E.g., missing semi-colons.

**Comment by JeroenDeDauw** [comment]
The backend HTML-building approach strikes again!

File: templates/ManageApprovers.mustache
Comment (waximabbax) [reply templates/ManageApprovers.mustache:1]: Sorry, my IDE messed up.

**Comment by waximabbax** [comment]
> The backend HTML-building approach strikes again!

Yeah. I wonder at what point it will force us to create separate Endpoints for POST. However, manipulating HTML with JS is also not very clean, but it is quite straightforward and doesn't require a lot of thinking.

File: templates/ManageApprovers.mustache (hunk 1 above)
Comment (waximabbax) [reply templates/ManageApprovers.mustache:33]: Screenshot from 2024-08-17 23-42-09

Comment (JeroenDeDauw) [reply templates/ManageApprovers.mustache:33]: `"`, not `?`

Comment (waximabbax) [reply templates/ManageApprovers.mustache:33]: @JeroenDeDauw " and ' also work fine.

Screenshot from 2024-08-18 12-52-30

**Pull request merged**